sdk/python/tests/integration/registration/test_cli.py

```diff
@@ -0,0 +1,75 @@
+import tempfile
+import uuid
+from pathlib import Path, PosixPath
+from textwrap import dedent
+
+import pytest
+import yaml
+
+from feast import FeatureStore, RepoConfig
+from tests.integration.feature_repos.repo_configuration import FULL_REPO_CONFIGS
+from tests.integration.feature_repos.universal.data_source_creator import (
+    DataSourceCreator,
+)
+from tests.utils.cli_utils import CliRunner, get_example_repo
+from tests.utils.online_read_write_test import basic_rw_test
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize("test_repo_config", FULL_REPO_CONFIGS)
+def test_universal_cli(test_repo_config) -> None:
```

Comment:
test_cli_local seems to have a lot of tests that check other commands. Do we want to add them here too?

Reply:
Yep good point

Reply:
Added them back.